apps/server/src/modules/legacy-school/legacy-school.module.ts

```diff
@@ -1,5 +1,5 @@
-import { GroupModule } from '@modules/group/group.module';
```

Comment:
@MarvinOehlerkingCap, this is not a solution to solve dependency cycles

Reply:
How should a solution look like?